components/hab/src/cli/hab/origin.rs

```diff
@@ -235,6 +240,51 @@ pub struct UploadGroup {
     public_file: Option<PathBuf>,
 }
 
+#[derive(ConfigOpt, StructOpt)]
+#[structopt(no_version, rename_all = "screamingsnake")]
+pub struct RbacShow {
+    #[structopt(flatten)]
+    pub origin:         BldrOrigin,
+    /// The account name of the role to display
+    pub member_account: String,
+    #[structopt(flatten)]
+    pub bldr_url:       BldrUrl,
+    #[structopt(flatten)]
+    pub auth_token:     AuthToken,
+    /// Output will be rendered in json
+    #[structopt(short = "j", long = "json")]
+    pub to_json:        bool,
+}
+
+#[derive(ConfigOpt, StructOpt)]
+#[structopt(no_version, rename_all = "screamingsnake")]
+pub struct RbacSet {
+    #[structopt(flatten)]
+    pub origin:         BldrOrigin,
+    /// The account name whose role will be changed
+    pub member_account: String,
+    /// The role name to enforce for the member account [values: readonly_member, member,
+    /// maintainer, administrator, owner]
```

Comment:
We could use `#[structopt(possible_values = &[OriginMemberRole::READONLY_MEMBER ...])]` which will automatically display the possible values and has better traceability should we ever add an OriginMemberRole` variant.

Reply:
This is a nice capability. Added in https://github.com/habitat-sh/habitat/pull/7841/commits/78f9d5e6a2b76e641bb6f2a2971c62744b5e762b